shared/src/main/scala/org/jetbrains/sbt/structure/data.scala

```diff
@@ -108,7 +111,7 @@ case class ConfigurationData(id: String,
 
 case class DirectoryData(file: File, managed: Boolean)
 
-case class JavaData(home: Option[File], options: Seq[String])
+case class JavaData(home: Option[File], options: Map[Configuration, Seq[String]])
```

Comment:
I could be helpful to extract `Map[Configuration, Seq[String]]` to a separate class and use it in all places (2?)

Reply:
What I actually had in mind is not replacing `Map` with `Seq` but extracting the whole type in a separate class.
With the Seq you now duplicate all the `configuration` values.
Map structure actually makes sense.
The original idea was to avoid duplication for `Map[Configuration, Seq[String]]` and have some name representing the data structure semantics.
The simplest approach could be just type alias:
```scala
type CompilerOptionsMapping = Map[Configuration, Seq[String]]
```
but a better-typed approach would be just using a wrapper class:
```scala
case class CompilerOptionsMapping(map: Map[Configuration, Seq[String]])
```

Reply:
> With the Seq you now duplicate all the configuration values.

How exactly the configurations are duplicated in the Seq?

Reply:
hmm but my class looks like that 
`case class CompilerOptions(configuration: Configuration, options: Seq[String])`
for each configuration (`conf1 `or `conf2`) a single instance of `CompilerOptions ` is created, so there are no repeats

Reply:
I did it like that because e.g. `ConfigurationData` is done in an analogous way

Reply:
My bad. Didn't look well.